src/2018/transitioning/ownership-and-lifetimes/anonymous-lifetime.md

```diff
@@ -79,6 +84,8 @@ impl<'a, 'b: 'a> Foo<'a, 'b> {
 We can rewrite this as:
 
 ```rust
+// Rust 2018
+
 #![feature(rust_2018_preview)]
```

Comment:
The comment and the feature gate need to be swapped lexically to make this pass the test, at least for now.